apps/hash-ai-worker-ts/src/activities/flow-activities/research-entities-action/link-follower-agent/get-link-follower-next-tool-calls.ts

```diff
@@ -0,0 +1,317 @@
+import type { Subtype } from "@local/advanced-types/subtype";
+import dedent from "dedent";
+
+import { getFlowContext } from "../../../shared/get-flow-context";
+import { getLlmResponse } from "../../../shared/get-llm-response";
+import type { LlmUserMessage } from "../../../shared/get-llm-response/llm-message";
+import { getToolCallsFromLlmAssistantMessage } from "../../../shared/get-llm-response/llm-message";
+import type {
+  LlmParams,
+  LlmToolDefinition,
+} from "../../../shared/get-llm-response/types";
+import { graphApiClient } from "../../../shared/graph-api-client";
+import type { LocalEntitySummary } from "../../shared/infer-facts-from-text/get-entity-summaries-from-text";
+import type { Fact } from "../../shared/infer-facts-from-text/types";
+import type { Link } from "./extract-links-from-content";
+
+const defaultModel: LlmParams["model"] = "claude-3-5-sonnet-20240620";
+
+const getLinkFollowerNextToolCallsSystemPrompt = dedent(`
+  You are a link follower agent.
+
+  <UserMessage>
```

Comment:
I think this could be confused with 'the message from the user' so I would rename it... not sure to what exactly... maybe `TaskInputsDescription` or something?

Reply:
Hmm I see what you mean, maybe `UserMessageDefinition`? Not sure what impact it has, but I feel it's a good idea to make it clear we're defining the contents of the user message. Have changed it in https://github.com/hashintel/hash/pull/4649/commits/65a1118f553bd2359c684201031cc4284621a35c

Reply:
Sounds good